packages/lesswrong/components/payments/AdminPaymentsPage.tsx

```diff
@@ -76,7 +75,7 @@ export const AdminPaymentsPage = ({classes}: {
 
   const currentUser = useCurrentUser()
   if (!currentUser?.isAdmin) return <SingleColumnSection>
-    <Typography variant='body2'>You don't have permission to view this page</Typography>
+    You don't have permission to view this page
```

Comment:
No @Raemon, you want to destructure it off of components. Don't give us times new roman.

Reply:
Replaced this with the standard component